velox/substrait/SubstraitToVeloxPlan.cpp

```diff
@@ -1588,21 +1588,34 @@ void SubstraitVeloxPlanConverter::setFilterMap(
   // Extract the column index and column bound from the scalar function.
   std::optional<uint32_t> colIdx;
   std::optional<::substrait::Expression_Literal> substraitLit;
+  std::vector<std::string> typeCases;
   for (const auto& param : scalarFunction.arguments()) {
     auto typeCase = param.value().rex_type_case();
     switch (typeCase) {
       case ::substrait::Expression::RexTypeCase::kSelection:
+        typeCases.emplace_back("kSelection");
         colIdx = subParser_->parseReferenceSegment(
             param.value().selection().direct_reference());
         break;
       case ::substrait::Expression::RexTypeCase::kLiteral:
+        typeCases.emplace_back("kLiteral");
         substraitLit = param.value().literal();
         break;
       default:
         VELOX_NYI(
             "Substrait conversion not supported for arg type '{}'", typeCase);
     }
   }
+
+  std::unordered_map<std::string, std::string> functionRevertMap = {
+      {"lt", "gt"}, {"gt", "lt"}, {"gte", "lte"}, {"lte", "gte"}};
```

Comment:
Maybe could be replaced by some variables at https://github.com/oap-project/velox/blob/main/velox/substrait/SubstraitToVeloxPlan.cpp#L66-L73.

Reply:
Updated.